README.md

```diff
@@ -136,19 +136,26 @@ We have left no HTTP verb behind. You can use these attributes on controller met
 
 ### Resource controllers
 
-To register a [resource controller](https://laravel.com/docs/controllers#resource-controllers), use the `Resource` attribute as shown in the example below.
+To register a [resource controller](https://laravel.com/docs/controllers#resource-controllers), use the `Resource` attribute as shown in examples below.
```

Comment:
Instead of referencing an example somewhere else, put the example immediately under this paragraph.